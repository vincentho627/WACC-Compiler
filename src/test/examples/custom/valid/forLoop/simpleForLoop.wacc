# For loop which should print "Change n" 5 times.

# Output:
# Change n
# Change n
# Change n
# Change n
# Change n

# Program:

begin
	for i in 0 until 5 do
		println "Change n"
	done
end
