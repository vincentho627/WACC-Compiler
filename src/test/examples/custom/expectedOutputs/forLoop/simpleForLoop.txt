Change n
Change n
Change n
Change n
Change n
